src/Evolution/Executables/Burgers/EvolveBurgers.hpp

```diff
@@ -62,17 +64,29 @@ struct EvolutionMetavars {
                  StepChoosers::Register::Constant,
                  StepChoosers::Register::Increase>;
 
+  using compute_rhs = tmpl::list<
```

Comment:
Is there an easy way for me to switch from GTS to LTS? Similarly for enabling/disabling self-starting? By easy I mean something like "comment this out, uncomment that". If so, should we document this somewhere? Maybe even just here as a C++ comment so Burgers can serve as an example for now until we actually have time to write real tutorials.

Reply:
is it possible to do something like the following (as I would rather change two bools than (un)comment multiple lines?
- define constexpr bools `use_local_timestepping` and `is_self_starting`
- breakup the current tmpl::lists into multiple pieces, and wrap them in a `tmpl::flatten`
- instead of commenting out line 72, say `tmpl::conditional_t<use_local_timestepping, tmpl::list<>, tmpl::list<dg::Actions::ApplyBoundaryFluxesGlobalTimeStepping>>`

Reply:
We already have a `local_time_stepping` bool, and using that sounds like a good idea.  Self starting is currently a runtime decision.